src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj

```diff
@@ -59,7 +59,7 @@
     <ProjectReference
       Include="..\testassets\BasicTestApp\BasicTestApp.csproj"
       Targets="Build;Publish"
-      Properties="TestTrimmedApps=true;PublishDir=$(MSBuildThisFileDirectory)$(OutputPath)trimmed\BasicTestApp\" />
+      Properties="BuildProjectReferences=false;TestTrimmedApps=true;PublishDir=$(MSBuildThisFileDirectory)$(OutputPath)trimmed\BasicTestApp\" />
```

Comment:
Do you want to do this for other ProjectReferences too?

Reply:
I'm not too sure if that would have any other ramifications. This is the specific project reference @rainersigwald pointed out based on the failing binlog.